### PR TITLE
Update add guest in user guide

### PR DESCRIPTION
**Pull request by bryanhce**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -87,21 +87,24 @@ Format: `help`
 
 ### Adding a guest: `add`
 
-Adds a guest to the guest book.
+Adds a guest to the guest book. This is akin to checking a guest in with their personal and room details.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]`
```
Comment (jehousoh) [inline docs/UserGuide.md]: Should we use PHONE instead of PHONE_NUMBER for consistency below?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -87,21 +87,24 @@ Format: `help`
...
-[//]: # (But may be relevant in the future)
-[//]: # (<div markdown="span" class="alert alert-primary">:bulb: **Tip:**)
+Constraints of parameters:
 
-[//]: # (A guest can have any number of tags &#40;including 0&#41;)
-
-[//]: # (</div>)
+Unless stated otherwise, all fields should not be left blank.
+* `NAME` should only contain alphanumeric characters and spaces.
+* `PHONE_NUMBER` should only contain numbers, and it should be at least 3 digits long.
```
Comment (jehousoh) [inline docs/UserGuide.md]: Same here for PHONE_NUMBER.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -87,21 +87,24 @@ Format: `help`
...
 
-[//]: # (Commented out first as not relevant since we do not have optional fields)
-[//]: # (But may be relevant in the future)
-[//]: # (<div markdown="span" class="alert alert-primary">:bulb: **Tip:**)
+Constraints of parameters:
 
-[//]: # (A guest can have any number of tags &#40;including 0&#41;)
-
-[//]: # (</div>)
+Unless stated otherwise, all fields should not be left blank.
```
Comment (jehousoh) [inline docs/UserGuide.md]: Should this be indented to show that it is under the Constraints of parameters?

image

File: docs/UserGuide.md [hunk 4]
```diff
@@ -87,21 +87,24 @@ Format: `help`
...
+* `NAME` should only contain alphanumeric characters and spaces.
+* `PHONE_NUMBER` should only contain numbers, and it should be at least 3 digits long.
+* `EMAIL` should only contain alphanumeric characters and '@' character.
+* `ROOM` should only contain alphanumeric characters and a hyphen.
+* `DATE_RANGE` should follow the format dd/MM/yy - dd/MM/yy.
+* `NUMBER_OF_GUESTS` should only contain numbers, and it should be at least 1 and at most 4.
+* `REQUEST` is an optional field.
```
Comment (jehousoh) [inline docs/UserGuide.md]: Should we add that there can at most be 500 characters?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -221,14 +224,14 @@ _Details coming soon ..._
...
-| **List**             | `list`                                                                                                                                                                              |
-| **Help**             | `help`                                                                                                                                                                              |
+| Action               | Format, Examples                                                                                                                                                                               |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**              | `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]` <br> e.g., `add n/John Doe p/98765432 e/johnd@example.com rm/ 05-73 dr/19/05/20 - 24/05/22 ng/3`    |
```
Comment (jehousoh) [inline docs/UserGuide.md]: Just a thought should we add rq/ for the example as well, or no since it's optional?

Comment (jehousoh) [inline docs/UserGuide.md]: Same for this for PHONE_NUMBER

File: docs/UserGuide.md [hunk 6]
```diff
@@ -221,14 +224,14 @@ _Details coming soon ..._
...
+| **Bill**             | `bill INDEX b/BILL`<br> e.g., `bill 2 b/99.99`                                                                                                                                                 |
+| **Find**             | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                     |
+| **MarkRoomsUnclean** | `markRoomsUnclean`                                                                                                                                                                             |
+| **List**             | `list`                                                                                                                                                                                         |
+| **Help**             | `help`                                                                                                                                                                                         |
```
Comment (jehousoh) [inline docs/UserGuide.md:237]: Should we add exit here as well?

**Review by jehousoh — CHANGES_REQUESTED** [review]
Some minor adjustments and considerations but good work!

File: docs/UserGuide.md (hunk 1 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Yes, edited!

File: docs/UserGuide.md (hunk 2 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Edited!

File: docs/UserGuide.md (hunk 3 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Edited!

File: docs/UserGuide.md (hunk 4 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Yes, nice catch!

File: docs/UserGuide.md (hunk 5 above)
Comment (bryanhce) [reply docs/UserGuide.md]: I added an example of the rq/ in the example under "adding a guest", maybe we can leave it out here since its optional?

Comment (bryanhce) [reply docs/UserGuide.md]: Edited!

File: docs/UserGuide.md (hunk 6 above)
Comment (bryanhce) [reply docs/UserGuide.md:237]: Yes, we should, but that part is left for Jerry to edit!

**Review by jehousoh — APPROVED** [review]
lgtm!

**Pull request merged**